recipes/apc.rb

```diff
@@ -26,9 +26,9 @@
   only_if { node['osl-php']['use_ius'] }
 end
 
-ruby_block 'raise_centos8_exception' do
+ruby_block 'raise_almalinux8_exception' do
```

Comment:
Let's just generalize this as `el` in case we move to something else down the road.
```suggestion
ruby_block 'raise_el8_exception' do
```